### PR TITLE
Update syntax for loginInBackground example

### PR DESCRIPTION
**Pull request by jaimeagudo**

### CODE REVIEW

**Comment by flovilmart** [comment]
The existing code should correctly work in swift. Why that change?

**Comment by nitrag** [comment]
@jaimeagudo Is correct. I would go as far as to include the error parsing, which was quite difficult to figure out on my own.

```
PFFacebookUtils.logInInBackground(withReadPermissions: ["public_profile","email"]) {
            (user: PFUser?, error: Error?) -> Void in
            if let error = error {
                if let errorString = (error as NSError).userInfo["error"] as? String {
                    NSLog(errorString);
                }
            } else {
                if let user = user {
                    if user.isNew {
                        print("User signed up and logged in through Facebook!")
                    } else {
                        print("User logged in through Facebook!")
                    }
                    self.currentUser = PFUser.current();
                    self.loggedIn()
                }
            }
        }
```

File: _includes/ios/users.md [hunk 1]
```diff
@@ -545,10 +545,10 @@ There are two main ways to use Facebook with your Parse users: (1) to log in (or
 }];
 </code></pre>
 <pre><code class="swift">
-PFFacebookUtils.logInInBackgroundWithReadPermissions(permissions) {
-  (user: PFUser?, error: NSError?) -> Void in
-  if let user = user {
-    if user.isNew {
+PFFacebookUtils.logInInBackground(withReadPermissions: permissions) {
+  (user: PFUser?, error: Error?) in
+  if let u = user {
```
Comment (flovilmart) [inline _includes/ios/users.md:550]: revert that change please

File: _includes/ios/users.md [hunk 2]
```diff
@@ -545,10 +545,10 @@ There are two main ways to use Facebook with your Parse users: (1) to log in (or
...
+    if u.isNew {
```
Comment (flovilmart) [inline _includes/ios/users.md:551]: that one too.

**Review by flovilmart — CHANGES_REQUESTED** [review]

File: _includes/ios/users.md (hunk 2 above)
Comment (jaimeagudo) [reply _includes/ios/users.md:551]: That's the current syntax and _Intellisense_ in Xcode is *&^#, why do you want to revert it?

Comment (flovilmart) [reply _includes/ios/users.md:551]: we prefer `let user = user` usually

**Comment by addisonElliott** [comment]
Was looking through the code, and I fail to see where logInInBackground is defined for PFFacebookUtils? Where is the code at?

**Comment by flovilmart** [comment]
@addisonElliott https://github.com/parse-community/Parse-SDK-iOS-OSX/blob/master/ParseFacebookUtils/ParseFacebookUtils/PFFacebookUtils.h#L74

**Comment by TomWFox** [comment]
As this is dormant I have merged #589 which fixes the syntax as suggested here with respect to the suggested changes from Flovilmart.

**Pull request closed without merging**